freqtrade/resolvers/strategy_resolver.py

```diff
@@ -24,8 +24,6 @@ class StrategyResolver(IResolver):
     This class contains all the logic to load custom strategy class
     """
 
-    __slots__ = ['strategy']
```

Comment:
Note to self: need to test this! reference of experiened error:

https://github.com/freqtrade/freqtrade/pull/2222#discussion_r321032785